roles/usegalaxy-eu.remove-orphan-condor-jobs/templates/remove-orphan-condor-jobs.j2

```diff
@@ -8,14 +8,16 @@ date
 
 IFS=$'\n'
 # Get running jobs from the HTCondor queue
-condor_running_jobs=(`condor_q -global -autoformat Cmd ClusterId JobStatus -json | jq -r -c '.[] | select(.JobStatus == 2) | [.Cmd, .ClusterId] | @sh' | cut -d " " -f1,2 | awk '{n = split($1, arr, "/"); print arr[n-1], $2}'`)
+condor_running_jobs=(`condor_q -global -autoformat Cmd ClusterId JobStatus -json | jq -r -c '.[] | select(.JobStatus == 2{% if remove_orphan_jobs_exclude %} and (.Cmd | test("{{ remove_orphan_jobs_exclude | map('regex_escape') | join('|') }}") | not){% endif %}) | [.Cmd, .ClusterId] | @sh' | cut -d " " -f1,2 | awk '{n = split($1, arr, "/"); print arr[n-1], $2}'`)
```

Comment:
maybe this is an imperfect approach and yours is probably working;
```suggestion
condor_running_jobs=(`condor_q -global -constraint 'regexp("(.*galaxy_[0-9]+\.sh)|(.*\/[0-9]+\/command\.sh)", Cmd)' -autoformat Cmd ClusterId JobStatus -json | jq -r -c '.[] | select(.JobStatus == 2 | [.Cmd, .ClusterId] | @sh' | cut -d " " -f1,2 | awk '{n = split($1, arr, "/"); print arr[n-1], $2}'`)
```

This way `condor_q` would filter for the "typical" galaxy job script pattern for embedded pulsar and "normal" jobs. Any other jobs would not be removed. 
Advantage would be a bit less code, disadvantage: if Galaxy's job script naming pattern changes in the future this might break.